mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/events/ElasticSearchListeningMessageSearchIndex.java

```diff
@@ -109,13 +109,24 @@ public ListenerType getType() {
     @Override
     public void add(MailboxSession session, Mailbox mailbox, MailboxMessage message) throws MailboxException {
         try {
+            LOGGER.info("Indexing mailbox {}-{} of user {} on message {}",
+                    mailbox.getName(), mailbox.getMailboxId(),
+                    session.getUser().getUserName(),
+                    message.getUid());
             indexer.indexMessage(indexIdFor(mailbox, message.getUid()), messageToElasticSearchJson.convertToJson(message, ImmutableList.of(session.getUser())));
         } catch (Exception e) {
             try {
-                LOGGER.warn("indexing message {} without attachments ", message.getUid());
+                LOGGER.warn("Indexing mailbox {}-{} of user {} on message {} without attachments ",
+                        mailbox.getName(), mailbox.getMailboxId(),
+                        session.getUser().getUserName(),
+                        message.getUid());
                 indexer.indexMessage(indexIdFor(mailbox, message.getUid()), messageToElasticSearchJson.convertToJsonWithoutAttachment(message, ImmutableList.of(session.getUser())));
             } catch (JsonProcessingException e1) {
-                LOGGER.error("Error when indexing message " + message.getUid() + " without its attachment", e1);
+                LOGGER.error("Error when indexing mailbox {}-{} of user {} on message {} without its attachment",
+                        mailbox.getName(), mailbox.getMailboxId(),
```

Comment:
I would use one argument one line. So please replace:
```
mailbox.getName(), mailbox.getMailboxId()
```
by
```
mailbox.getName(),
mailbox.getMailboxId()
```
It's more easy to modify, understand diff (in the futur) and count argument to check you put the correct number of place holder.